aiohttp/web_response.py

```diff
@@ -108,6 +109,7 @@ def __init__(
         self._keep_alive = None  # type: Optional[bool]
         self._chunked = False
         self._compression = False
+        self._compression_strategy: Optional[int] = None
```

Comment:
Seems like `None` is not a valid strategy. Why not set it to the same default (`zlib.Z_DEFAULT_STRATEGY`)?

Otherwise, maybe we could reuse the `self._compression` property. Define it as `Optional[int]` and then change the check in `_prepare_headers()` to `if self._compression is not None:`. Might need a little more tweaking to get it right though.

Reply:
I've changed `self._compression_strategy` from `None` to `zlib.Z_DEFAULT_STRATEGY` but it didn't help.
Reuse `self._compression`? But how it is related to `self._compression_strategy`?
I changed `strategy` parameter in `StreamWriter.enable_compression()` from `strategy: int = zlib.Z_DEFAULT_STRATEGY` to `strategy: Optional[int] = zlib.Z_DEFAULT_STRATEGY` and it did the trick.

Reply:
Sorry, `make lint` failed again. Finally I changed `Optional[int]` to `int` everywhere because linter says that `zlib.compressobj()` does not accept `Optional[int]`.
Now `mypy` test is being passed successfully for my changes